flink-tests/src/test/java/org/apache/flink/test/web/WebFrontendITCase.java

```diff
@@ -110,21 +112,18 @@ public void getTaskmanagers() {
 		}
 	}
 
-// TODO activate this test after logging retrieval has been added to the new web frontend
-//	@Test
-//	public void getLogfiles() {
-//		try {
-//			String logPath = cluster.configuration().getString(ConfigConstants.JOB_MANAGER_WEB_LOG_PATH_KEY, null);
-//			Assert.assertNotNull(logPath);
-//			FileUtils.writeStringToFile(new File(logPath, "jobmanager-main.log"), "test content");
-//
-//			String logs = getFromHTTP("http://localhost:8081/logInfo");
-//			Assert.assertTrue(logs.contains("test content"));
-//		}catch(Throwable e) {
-//			e.printStackTrace();
-//			Assert.fail(e.getMessage());
-//		}
-//	}
+	@Test
+	public void getLogAndStdoutFiles() {
+		try {
+			String logs = getFromHTTP("http://localhost:" + port + "/jobmanager/log");
+			Assert.assertTrue(logs.contains("job manager log"));
+			logs = getFromHTTP("http://localhost:" + port + "/jobmanager/stdout");
+			Assert.assertTrue(logs.contains("job manager out"));
```

Comment:
How about writing something to the log or stdout and checking for the result here?